src/clj/game/cards-operations.clj

```diff
@@ -312,6 +312,13 @@
             :label "Give the Runner 4 tags"
             :effect (effect (tag-runner :runner 4))}}
 
+   "Hatchet Job"
+   {:trace {:base 5
+            :choices {:req #(and (installed? %)
+                                 (not (has-subtype? % "Virtual")))}
+            :msg "add an installed non-virtual card to the Runner's grip"
+            :effect (effect (move :runner target :hand true))}}
```

Comment:
Not sure why there is `true` here at the end, but that's how Archangle does it...